zot.yaml

```diff
@@ -16,17 +16,44 @@ environment:
       - busybox
       - go
       - curl
+      - patch
 
 pipeline:
   - uses: git-checkout
     with:
       repository: https://github.com/project-zot/zot
       tag: v${{package.version}}
-      destination: zot
       expected-commit: 69f0cf6bb4727884af42f38c92858fdb114104de
 
   - runs: |
-      cd zot
+      # Resolve numerous vulnerabilities, and resolve dependency issues caused by these changes.
+
+      go mod edit -replace 'go.opentelemetry.io/otel=go.opentelemetry.io/otel@v1.12.0'
+      go mod edit -replace 'google.golang.org/grpc=google.golang.org/grpc@v1.54.1'
+
+      # Remove replace directives that don't appear to be necessary and introduce vulnerabilities.
+      sed -i '/github\.com\/opencontainers\/runc v1\.1\.2/d' go.mod
+      sed -i '/github\.com\/hashicorp\/go-getter v1\.6\.1/d' go.mod
+
+      go get github.com/docker/distribution@v2.8.2-beta.1
+      go get github.com/docker/docker@v20.10.24
+      go get github.com/hashicorp/go-getter@v1.7.0
+      go get github.com/opencontainers/runc@v1.1.5
+      go get github.com/sigstore/rekor@v1.2.2
+      go get golang.org/x/net@v0.11.0
+
+      go mod tidy
+
+  - runs: |
+      # Zot hasn't been updated to handle the latest version of the image-spec module.
+      # Revert a particular breaking commit in image-spec (63b8bd0) until zot does upgrade.
+
+      moduleDir="$(go env GOPATH)/pkg/mod/github.com/opencontainers/image-spec@v1.1.0-rc3"
+      chmod -R u+w "$moduleDir"
+      patch -p1 -d "$moduleDir" -N < ./revert-removal-of-symbols.patch
+      go clean -cache
```

Comment:
This reverts an upstream commit from @jonjohnsonjr 😁  — temporarily! We have reason to believe this won't be necessary for much longer.